tests/bats/test_local_mounts.bats

```diff
@@ -56,6 +56,8 @@ ${AIRFLOW_SOURCES}/NOTICE:/opt/airflow/NOTICE:cached
 -v
 ${AIRFLOW_SOURCES}/airflow:/opt/airflow/airflow:cached
 -v
+${AIRFLOW_SOURCES}/backport_packages/setup_backport_packages.py:/opt/airflow/backport_packages/setup_backport_packages.py:cached
```

Comment:
Off topic to this PR, but this test seems quite low value to me -- we just have to keep the list in sync in two places. A generic test to check the behavour of convert_local_mounts_to_docker_params I'm cool with, but having to update this test just when we add a new list: not so much. I don't see what it buys us?

Reply:
True. It's been useful when I had BASH YAML parser but the value is limited no. I will just let it run and not compare the results, just to count the number of lines.

Reply:
The test now checks if exactly half of the lines produced by the output is "-v".

Reply:
Is it possible to run it with a specific set of inputs, so the test output doesn't change?

Reply:
Yes, we could, by overridiing LOCAL_MOUNTS variable. I can do that indeed.